lib/rumoji/emoji/symbols.rb

```diff
@@ -75,7 +75,7 @@ class Emoji
       self.new("\u{1f564}", [:clock930]),
       self.new("\u{3297}" , [:congratulations]),
       self.new("\u{1f192}", [:cool]),
-      self.new("\u{00a9}" , [:copyright]),
```

Comment:
I assume you noted that I commented these out?  We didn't like it replacing normal, non-emoji characters.